File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/ocso/OCSACTGetFindWorkerInfo.java
package nta.med.data.model.ihis.ocso;

public class OCSACTGetFindWorkerInfo {
	private String ordDanui ;
	private String codeName ;
	private Double seq       ;
	public OCSACTGetFindWorkerInfo(String ordDanui, String codeName, Double seq) {
		super();
		this.ordDanui = ordDanui;
		this.codeName = codeName;
		this.seq = seq;
	}
	public String getOrdDanui() {
		return ordDanui;
	}
	public void setOrdDanui(String ordDanui) {
		this.ordDanui = ordDanui;
	}
	public String getCodeName() {
		return codeName;
	}
	public void setCodeName(String codeName) {
		this.codeName = codeName;
	}
	public Double getSeq() {
		return seq;
	}
	public void setSeq(Double seq) {
		this.seq = seq;
	}     
	

}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Ocso/OCSACTGetFindWorkerInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Ocso
{
    [Serializable]
    public class OCSACTGetFindWorkerInfo
    {
        private String _ordDanui;
        private String _codeName;
        private String _seq;

        public String OrdDanui
        {
            get { return this._ordDanui; }
            set { this._ordDanui = value; }
        }

        public String CodeName
        {
            get { return this._codeName; }
            set { this._codeName = value; }
        }

        public String Seq
        {
            get { return this._seq; }
            set { this._seq = value; }
        }

        public OCSACTGetFindWorkerInfo() { }

        public OCSACTGetFindWorkerInfo(String ordDanui, String codeName, String seq)
        {
            this._ordDanui = ordDanui;
            this._codeName = codeName;
            this._seq = seq;
        }

    }
}